### ISSUE TITLE
[CMIP6 CMOR-ization & ESGF-publication] NorESM2-LM - esm-ssp534-over

### ISSUE THREAD

**Issue by JorgSchwinger** [NorESMhub/noresm2cmor#339]
### Mandatory information:

**Full path to the case(s) of the experiment on NIRD**
/projects/projects/NS9560K/noresm/casesNSSP534frc2esm_f19_tn14_20230106

**experiment_id**
esm-ssp534-over 

**model_id**
NorESM2-LM

**CASENAME(s) and years to be CMORized**
NSSP534frc2esm_f19_tn14_20230106, 2015-2100
NSSP534frc2esmext_f19_tn14_20230510, 2101-2299

### Optional information
(additional information, if the experiment is branched/hybrid restart from previous parent experiment; you may find more information relevant to the experiment_id here: https://github.com/NorESMhub/noresm2cmor/blob/master/tables/CMIP6_CV.json)

**parent_experiment_id**
<strike>esm-hist</strike>
esm-ssp585

**parent_experiment_rip**
r1i1p1

**parent_time_units**

**branch_method**
<strike>Hybrid-restart from year 2015-01-01 of esm-hist</strike>
<strike>Hybrid-restart from year 2015-01-01 of esm-585</strike>
Hybrid-restart from year 2015-01-01 of esm-hist

**other information**
(provide other information that might be useful)
realization 1 (r1i1p1)

Some project partners are waiting to use these data for a publication, so if it is possible please prioritize this request (thanks!)

**Comment by YanchunHe** [comment]
The `parent_experiment_id` should be `esm-ssp585` according to the definition in `CMIP6_CV.json` .

I changed to esm-ssp585, if this is intended.

**Comment by JorgSchwinger** [comment]
Although it is possible to branch this experiment off esm-ssp585 in 2040, we did not do this in this case. We started the simulation in 2015 as a hybrid restart from esm-hist as originally stated. If you look at ESGF, both option have been dome by different groups so it should be okay.

**Comment by monsieuralok** [comment]
@YanchunHe published

**Comment by adagj** [comment]
Hi,
The precipitation data seems to be missing for years 2015-2099. Can we add those? @YanchunHe

**Comment by adagj** [comment]
Yes, it is missing but there are PRECC and PRECL and the sum should equal PRECT I think. @oyvindseland

**Comment by oyvindseland** [comment]
Yes, the sum equals PRECT.

**Comment by YanchunHe** [comment]
precipitation data is cmorized and ready to be published to ESGF.

The new `pr` dataset with `v20250113` covers 2015-2299. Therefore, there will be old version `v20230616` for 2100-2299.  If possible, the uncomplete version of v20230616 can be retracted, otherwise, both will be kept.

@monsieuralok could you please publish it.

## data path
* /projects/NS9034K/CMIP6/.cmorout/NorESM2-LM/esm-ssp534-over
* /projects/NS9034K/CMIP6/CDRMIP/NCC/NorESM2-LM/esm-ssp534-over

## version
* v20250113

## sha256sum
/projects/NS9034K/CMIP6/CDRMIP/NCC/NorESM2-LM/esm-ssp534-over
* .r1i1p1f1.sha256sum_v20250113

**Comment by monsieuralok** [comment]
@YanchunHe published
